extensions/amp-nested-menu/0.1/test/validator-amp-nested-menu-error.out

```diff
@@ -0,0 +1,87 @@
+|      <!-- Invalid: incorrect side value -->
+|      <amp-nested-menu layout="fill" side="center">
+>>     ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:36:4 The attribute 'side' in tag 'amp-nested-menu' is set to the invalid value 'center'. (see https://amp.dev/documentation/components/amp-nested-menu)
+|        <!-- Invalid: no element can include more than one submenu attributes -->
+|        <div>
+|          <h4 amp-nested-submenu-open amp-nested-submenu-close></h4>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:39:8 Mutually exclusive attributes encountered in tag 'h4 amp-nested-menu' - pick one of ['amp-nested-submenu-close', 'amp-nested-submenu-open'].
+|          <div amp-nested-submenu amp-nested-submenu-close></div>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:40:8 Mutually exclusive attributes encountered in tag 'div amp-nested-menu' - pick one of ['amp-nested-submenu', 'amp-nested-submenu-close', 'amp-nested-submenu-open'].
+|          <div amp-nested-submenu amp-nested-submenu-open></div>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:41:8 Mutually exclusive attributes encountered in tag 'div amp-nested-menu' - pick one of ['amp-nested-submenu', 'amp-nested-submenu-close', 'amp-nested-submenu-open'].
+|        </div>
+|        <!-- Invalid: only divs may include the amp-nested-submenu attribute -->
+|        <div>
+|          <span amp-nested-submenu></span>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:45:8 The attribute 'amp-nested-submenu' may not appear in tag 'span'.
+|          <button amp-nested-submenu></button>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:46:8 The attribute 'amp-nested-submenu' may not appear in tag 'button'.
+|        </div>
+|        <!-- Invalid: only buttons, divs, spans and h2-h6 may have submenu open/close attribute -->
+|        <div>
+|          <a amp-nested-submenu-open></a>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:50:8 The attribute 'amp-nested-submenu-open' may not appear in tag 'a'. (see https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml#links)
+|          <h1 amp-nested-submenu-close></h1>
+>>         ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:51:8 The attribute 'amp-nested-submenu-close' may not appear in tag 'h1'.
+|        </div>
+|        <!-- Invalid: submenus cannot be inside an amp-accordion -->
+|        <amp-accordion>
+|          <section>
+|            <h4>accordion header</h4>
+|            <div>
+|              <h4 amp-nested-submenu-open></h4>
+|              <div amp-nested-submenu></div>
+>>             ^~~~~~~~~
+amp-nested-menu/0.1/test/validator-amp-nested-menu-error.html:59:12 The attribute 'amp-nested-submenu' may not appear in tag 'div'.
```

Comment:
This is an unfortunate error message since it really should be reporting the issue is it being a descendant of amp-accordion. We should file an issue against the validator on error selection once this is submitted so the issue can point it out.

Reply:
Filed https://github.com/ampproject/amphtml/issues/25499. Thank you!